crates/typst-library/src/layout/grid/mod.rs

```diff
@@ -498,8 +498,8 @@ pub struct GridFooter {
 pub struct GridHLine {
     /// The row above which the horizontal line is placed (zero-indexed).
     /// If the `position` field is set to `{bottom}`, the line is placed below
-    /// the row with the given index instead (see that field's docs for
-    /// details).
+    /// the row with the given index instead (see
+    /// [`grid.hline.position`]($grid.hline.position) for details).
```

Comment:
I think you don't even need the `(..)` part as it defaults to what's in the backticks.

Reply:
If I understood you correct, it should be sufficient to write this?

```typst
(see  [`grid.hline.position`] for details).
```

If so, it seems like the
```typst
[`foo.bar`]($foo.bar)
````
pattern is quite pervasive throughout the comments. E.g.

from https://github.com/typst/typst/blob/b98954c78b0c9247db0a669838872227dbdd59e4/crates/typst-library/src/layout/grid/mod.rs#L65
```typst
[`grid.cell`]($grid.cell)
```

Maybe I misunderstood something? : )

Reply:
Seems like the grid documentation also unnecessarily specifies this.